unity/MexRestSample/Assets/Plugins/iOS/PlatformIntegration.m

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright 2019 MobiledgeX, Inc. All rights and licenses reserved.
```

Comment:
Date 2020.

Reply:
Shall I change this date on all of the scripts on the MexRestSample and PingPong?

Reply:
I think we are doing 2018-2020 @lgarner

Reply:
If there are a lot of changes, that can be a separate PR